lib/agent.js

```diff
@@ -369,6 +419,19 @@ Agent.prototype._handleMessage = function(request, callback) {
         return this._fetchSnapshot(request.c, request.d, request.v, callback);
       case 'nt':
         return this._fetchSnapshotByTimestamp(request.c, request.d, request.ts, callback);
+      case 'p':
+        var presence = this._createPresence(request);
```

Comment:
Let's hide this behind a feature flag for now. Will need to update the docs, saying that it's behind a flag and sort of beta-y, with potential for fan-out issues with lots of presence clients.